Package.swift

```diff
@@ -56,6 +56,9 @@ let package = Package(
       name: "GRPC",
       dependencies: [
         .product(name: "NIO", package: "swift-nio"),
+        .product(name: "NIOCore", package: "swift-nio"),
+        .product(name: "NIOPosix", package: "swift-nio"),
+        .product(name: "NIOEmbedded", package: "swift-nio"),
```

Comment:
I know that this is to setup a client for a throughput test, but linking `NIOEmbedded` looks wrong :) maybe we can change this with a follow up PR.